### PR TITLE
REST > Missing 2 chargingstations endpoints

### PR DESCRIPTION
**Pull request by Ninjeneer**
2 endpoints were missing :
- Smart charging trigger (GET /chargingstations/smartcharging/trigger)
- Charginstation OCPP configuration update (PUT /chargingstations/{id}/configuration)

Also made minor improvements on doc parameters

### CODE REVIEW

File: src/assets/server/rest/v1/docs/e-mobility-oas.json [hunk 1]
```diff
@@ -2645,7 +2679,7 @@
             "$ref": "#/components/parameters/SiteID"
           },
           {
-            "$ref": "#/components/parameters/SiteAreaID"
+            "$ref": "#/components/parameters/SiteAreaIDs"
```
Comment (LucasBrazi06) [inline src/assets/server/rest/v1/docs/e-mobility-oas.json:2682]: SiteAreaID should always be singular like any other query params where we can pass several values that are separated by a pipe.

**Review by LucasBrazi06 — CHANGES_REQUESTED** [review]

File: src/assets/server/rest/v1/docs/e-mobility-oas.json (hunk 1 above)
Comment (Ninjeneer) [reply src/assets/server/rest/v1/docs/e-mobility-oas.json:2682]: Actually, the Query param's name itself is not modified. I just changed the name of the param in the oas file to differentiate Single ID / Multiple IDs.
As the TriggerSmartCharging endpoint focus on one SiteAreaID, we should not use a param labeled as "Pipe separated IDs"

Comment (LucasBrazi06) [reply src/assets/server/rest/v1/docs/e-mobility-oas.json:2682]: > Actually, the Query param's name itself is not modified. I just changed the name of the param in the oas file to differentiate Single ID / Multiple IDs.
> As the TriggerSmartCharging endpoint focus on one SiteAreaID, we should not use a param labeled as "Pipe separated IDs"

Okay got it, then we should also align the naming of all the other query params ;-)

Comment (Ninjeneer) [reply src/assets/server/rest/v1/docs/e-mobility-oas.json:2679]: Indeed! Done

**Review by LucasBrazi06 — APPROVED** [review]

**Pull request merged**